keepassxc-browser/content/fields.js

```diff
@@ -92,9 +92,12 @@ kpxcFields.getSegmentedTOTPFields = function(inputs, combinations) {
     if (!kpxc.settings.showOTPIcon) {
         return;
     }
-    const addTotpFieldsToCombination = function(inputFields) {
+
+    let exceptionFound = false;
+
+    const addTotpFieldsToCombination = function(inputFields, ignoreLength = false) {
         const totpInputs = Array.from(inputFields).filter(e => e.nodeName === 'INPUT' && e.type !== 'password' && e.type !== 'hidden');
-        if (totpInputs.length === 6) {
+        if (totpInputs.length === 6 || ignoreLength) {
```

Comment:
How about instead we add a "expectedDigits" variable and default its value to 6.

Reply:
Good idea.

Reply:
Ok not too concerned, you can merge